azure-pipelines.yml

```diff
@@ -243,3 +261,11 @@ stages:
             cp -v MoarVM-cover/merge-profraw.sh ../nqp/ &&
             ./html-cover.sh 2
           condition: and(succeeded(), eq( variables['COVERAGE'], 'yes' ))
+          displayName: Create coverage report
+
+        - script: |
+            valgrind --leak-check=full ../install/bin/raku --full-cleanup -e '' 2>valgrind.log
+            cat valgrind.log
+            ../install/bin/raku -ne 'BEGIN my @lost; @lost.push($0) if /"lost: " (\d+)/; END exit @lost.elems != 3 || all(@lost) != 0' valgrind.log
```

Comment:
To make sure I understand this correctly: Three entries with each 0 lost bytes is expected?

Reply:
Yep, exactly correct. Looks like this:
```
==826695==    definitely lost: 0 bytes in 0 blocks
==826695==    indirectly lost: 0 bytes in 0 blocks
==826695==      possibly lost: 0 bytes in 0 blocks
```

Reply:
OK. Thanks for the explanation!